internal/testutils/bubblewrap.go

```diff
@@ -70,16 +74,35 @@ func RunTestInBubbleWrap(t *testing.T, args ...string) {
 	args = append(args, testCommand...)
 
 	t.Logf("Running %s in bubblewrap", t.Name())
-	err := runInBubbleWrap(t, bubbleWrapNeedsSudo, "", nil, args...)
+	err := runInBubbleWrap(t, bubbleWrapNeedsSudo, nil, args...)
 	if err != nil {
 		t.Fatalf("Running %s in bubblewrap failed: %v", t.Name(), err)
 	}
 }
 
-func runInBubbleWrap(t *testing.T, withSudo bool, testDataPath string, env []string, args ...string) error {
+func runInBubbleWrap(t *testing.T, withSudo bool, env []string, args ...string) error {
 	t.Helper()
 
-	cmd := exec.Command("bwrap")
+	// Since 25.10 Ubuntu ships the AppArmor profile /etc/apparmor.d/bwrap-userns-restrict
+	// which restricts bwrap and causes chown to fail with "Operation not permitted".
+	// We work around that by copying the bwrap binary to a temporary location so that
+	// the AppArmor profile is not applied.
```

Comment:
Wow... Well definitely a quite relevant apparmor flaw I'd say... Why does it even has apparmor profiles? :)

Reply:
> as otherwise an app could just replicate the same done here

to do what we're doing here an app already needs privileges which allow it to execute arbitrary code as the current process (i.e. the process which launches bwrap). I really don't see an issue here.